adapter-excel/src/main/java/ch/prevo/open/node/adapter/excel/CommencementNotificationWriter.java

```diff
@@ -18,16 +18,11 @@
  ******************************************************************************/
 package ch.prevo.open.node.adapter.excel;
 
-import ch.prevo.open.data.api.FullCommencementNotification;
+import ch.prevo.open.data.api.CapitalTransferInformation;
 import ch.prevo.open.data.api.EmploymentInfo;
+import ch.prevo.open.data.api.FullCommencementNotification;
 import ch.prevo.open.encrypted.model.Address;
-import ch.prevo.open.encrypted.model.CapitalTransferInformation;
-import org.apache.poi.ss.usermodel.Cell;
-import org.apache.poi.ss.usermodel.CellStyle;
-import org.apache.poi.ss.usermodel.Font;
-import org.apache.poi.ss.usermodel.Row;
-import org.apache.poi.ss.usermodel.Sheet;
-import org.apache.poi.ss.usermodel.Workbook;
+import org.apache.poi.ss.usermodel.*;
```

Comment:
can we agree on not using star-imports?

Reply:
I prefer them (indicate coarser-level dependencies and can actually be read), but can be more conservative with organizing imports if this is important for the others.